src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double sideA;
+    private double sideB;
+    private double sideC;
+
+    public IsoscelesTrapezoid(double sideA, double sideB, double sideC, String color) {
```

Comment:
It's better to keep same structure of constructor arguments. Sometimes you have color as first argument, sometimes as last argument. Make it first or last (same) in all constructors